### PR TITLE
Only communicate the next tick of activation of EV simulation as a whole

### LINKED ISSUE #176: Only communicate the next tick of activation of EV simulation as a whole
Per https://github.com/ie3-institute/MobilitySimulator/issues/254#issuecomment-2064255520, communication of next ticks has limitations that only allows to communicate the next tick at which the EV simulation will be activated as a whole, not per EVCS. (#171 needs to be reverted as part of this.)

### PR DESCRIPTION
**Pull request by sebastian-peter**
Resolves #176

### CODE REVIEW

File: src/main/java/edu/ie3/simona/api/data/ev/ontology/ProvideArrivingEvs.java [hunk 1]
```diff
@@ -6,14 +6,18 @@
...
 import java.util.Map;
+import java.util.Optional;
 import java.util.UUID;
 
 /**
- * Provide arriving EVs data to SIMONA and its charging stations
+ * Provide arriving EVs to SIMONA and its charging stations
  *
- * @param arrivingEvsData the arriving EVs data per charging station UUID
+ * @param arrivals the arriving EVs per charging station UUID
```
Comment (jo-bao) [inline src/main/java/edu/ie3/simona/api/data/ev/ontology/ProvideArrivingEvs.java:18]: same as above

**Review by jo-bao — APPROVED** [review]
So for me it's fine beside two small remarks (but maybe it's personal preference)

File: src/main/java/edu/ie3/simona/api/data/ev/ExtEvData.java [hunk 2]
```diff
@@ -82,10 +81,12 @@ public List<EvModel> requestDepartingEvs(Map<UUID, List<UUID>> departures)
    * Provide all EVs that are arriving at some charging station to SIMONA. Method returns right away
    * without expecting an answer from SIMONA.
    *
-   * @param arrivals the arriving EVs data per charging station UUID
+   * @param arrivals the arriving EV models per charging station UUID
```
[the remark this answers is not in the document]
Comment (sebastian-peter) [reply src/main/java/edu/ie3/simona/api/data/ev/ExtEvData.java:84]: @jo-bao What's strange about it?

[the remark this answers is not in the document]
Comment (sebastian-peter) [reply src/main/java/edu/ie3/simona/api/data/ev/ExtEvData.java:84]: Confusion was lifted after short talk with @jo-bao

**Pull request merged**